public_html/quickstart.php

```diff
@@ -264,7 +265,8 @@
 						<br>
 						<i>Note: The PlayStation 3 has a maximum file size of 4GB. When dumping disc-based games which contain files bigger than 4GB, multiMAN will split those files. When you have your dump over on your computer, you must rejoin the split files back together, otherwise the dump won't work.</i><br>
 						<br>
-						 For dumping digital games, you must copy the game folder from dev_hdd0/game/GameID on your console over to the same path on your RPCS3 folder. You also need to get your console's IDPS, the game's ACT.DAT and RIF, in order to generate a .RAP license file to be used in the emulator.
+						 For dumping digital games, you must copy the game folder from dev_hdd0/game/GameID on your console over to the same path on your RPCS3 folder. You also need to get your console's IDPS, the game's ACT.DAT and RIF, in order to generate a .RAP license file to be used in the emulator.<br>

```

Comment:
I know it wasn't here before as well, but all file paths seems to be highlighted. Consider highlighting it for consistency.